src/common/models/data-cube/data-cube.ts

```diff
@@ -238,6 +240,7 @@ export class DataCube implements Instance<DataCubeValue, DataCubeJS> {
   static DEFAULT_DEFAULT_FILTER = EMPTY_FILTER;
   static DEFAULT_DEFAULT_SPLITS = EMPTY_SPLITS;
   static DEFAULT_DEFAULT_DURATION = Duration.fromJS("P1D");
+  static DEFAULT_MAX_SPLITS = 3;
```

Comment:
3 is quite low number for me as for the max splits size. 5 or 6 is more realistic.

Reply:
More than two splits is only possible on Table and there, plywood generates a lot of top-ns. For 4 splits and reasonable limits we could get more than 500 druid queries. If admin is aware of risk, this value can be overwritten in data cube configuration.